lib/known_bugs.pm

```diff
@@ -48,7 +48,12 @@ sub create_list_of_serial_failures {
     }
 
 
-    push @$serial_failures, {type => 'hard', message => 'CPU soft lockup detected', pattern => quotemeta 'soft lockup - CPU'};
+    if (check_var('BACKEND', 'qemu')) {
+        push @$serial_failures, {type => 'soft', message => 'CPU soft lockup detected', pattern => quotemeta 'soft lockup - CPU'};
+    }
+    else {
+        push @$serial_failures, {type => 'hard', message => 'CPU soft lockup detected', pattern => quotemeta 'soft lockup - CPU'};
```

Comment:
I would not duplicate the whole line just to differ by the type. However, I don't see why we should differ by the backend at all anyway

Reply:
@okurz , with snapshots, qemu is different from real hardware tests and thus, can trigger some bug detection where there is no bug.

Reply:
Outdated with latest update.